setup.md

```diff
@@ -16,3 +16,52 @@ $ cd
 {: .bash}
 
 In the lesson, you will find out how to access the data in this folder.  
+
+> ## Where to type commands: How to open a new shell
+> The `shell` is a program that enables us to send commands to the computer and recive output. It is also referred to as the `terminal` or `command line`.
+>
+> Some computers include a default Unix Shell program. 
+> The steps below describe some methods for identifying and opening a Unix Shell program if you already have one installed. 
+> There are also options for identifying and downloading a Unix Shell program, a Linux/UNIX emulator, or a program to access a UNIX server. 
+>
+> If none of the options below address your circumstances, try an online search for: UNIX shell [your computer model] [your operating system].
+>
+> ### Linux
+> The default shell for Linux operating systems is usually Bash.
+> On most versions of Linux, it is accessible by running the Terminal program,
+>  which can be found via the applications menu or the search bar.  
+> If your machine is set up to use something other than bash, you can run it by opening a terminal and typing `bash`.
+>
+> ### Mac OS
+> For a Mac computer, the default Unix Shell is Bash,
+> and it is available via the Terminal Utilities program within your Applications folder.
+>
+> To open Terminal, try one or both of the following:
+> * Go to your Applications. Within Applications, open the Utilities folder. Locate Terminal in the Utilities folder and open it.
+> * Use the Mac ‘Spotlight’ computer search function. Search for: Terminal and press [Enter] - this will open Terminal.
+>
+> #### Reference 
+> [How to Use Terminal on a Mac](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/)
+>
+> ### Windows
+> Computers with Windows operating systems do not automatically have a Unix Shell program installed.
+> In this lesson, we encourage you to use an emulator included in Git for Windows. 
+> This gives you access to both Bash shell commands and Git. 
+>
```

Comment:
Can you add a note here that if you're attending a SWC session, its likely you received instructions for this.